src/app/public/modules/inline-delete/inline-delete.component.scss

```diff
@@ -0,0 +1,39 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+
+.sky-inline-delete {
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, .5);
+  z-index: 999;
+  border: 2px solid $sky-highlight-color-danger;
+}
+
+.sky-inline-delete-assistive-text {
+  @include sky-screen-reader-only();
+}
+
+.sky-inline-delete-button {
+  margin-right: $sky-margin;
+}
+
+.sky-inline-delete-card .sky-inline-delete-content {
```

Comment:
Just curious - Why did you decide to handle the different styles from the inline-delete component, instead of using some nested classes in the consuming component. For example, the card component could have some styles like...
```
.sky-card .sky-inline-delete {
// card-specific inline-delete styles
}
```

Reply:
Got it. I'm happy with this then!